system_tests/logging_.py

```diff
@@ -175,3 +177,25 @@ def test_create_sink_bigquery_dataset(self):
         sink.create()
         self.to_delete.append(sink)
         self.assertTrue(sink.exists())
+
+    def test_create_sink_pubsub_topic(self):
+        from gcloud import pubsub
+        TOPIC_URI = 'pubsub.googleapis.com/projects/%s/topic/%s' % (
+            Config.CLIENT.project, TOPIC_NAME)
```

Comment:
That is *definitely* not clearer than the current code.  Not only that, but it relies on the accident that the `netloc` of the `api_base_url` happens to be the string specified in the `projects.sinks.create` API.